pkgs/development/python-modules/check-manifest/default.nix

```diff
@@ -14,9 +24,20 @@ buildPythonPackage rec {
     substituteInPlace tests.py --replace "test_build_sdist" "no_test_build_sdist"
   '';
 
-  propagatedBuildInputs = [ build pep517 toml ];
```

Comment:
I found it surprising to see that `build` is a runtime dependency. So `check-manifest` is a build-time dependency only, hence it makes more sense it executes `build`. `check-manifest` should typically be in `nativeBuildInputs` in other expressions.